openhands/runtime/impl/docker/docker_runtime.py

```diff
@@ -293,6 +294,8 @@ def _init_container(self):
             self.container = self.docker_client.containers.run(
                 self.runtime_container_image,
                 command=command,
+                # Override the default 'bash' entrypoint because the command is a binary.
+                entrypoint=[],
```

Comment:
Oddly this specifically is needed in Ubuntu when the default `ubuntu` user has been removed - but the updated config seems more correct all around.